### ISSUE TITLE
@package tag missing from check-tag-names

### ISSUE THREAD

**Issue by cloewen8** [gajus/eslint-plugin-jsdoc#163]
The @package tag is not checked for.

With `"jsdoc/check-tag-names": "error"`, the following shows an error:
```js
/**
 * @package
 */
export function func() {}
```
The above code should not produce any errors.
